src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -130,16 +130,23 @@ public override void OnStopActivity(Activity activity, object payload)
                     return;
                 }
 
-                var response = context.Response;
-                activity.AddTag(SpanAttributeConstants.HttpStatusCodeKey, response.StatusCode.ToString());
+                if (this.TryGetGrpcMethod(activity, out var grpcMethod))
+                {
+                    this.AddGrpcAttributes(activity, grpcMethod, context);
+                }
+                else
+                {
+                    var response = context.Response;
+                    activity.AddTag(SpanAttributeConstants.HttpStatusCodeKey, response.StatusCode.ToString());
```

Comment:
Not certain I've made the right decision here. Does it make sense to have a notion of both HTTP status and gRPC status?

Also, `OnStartActivity` also adds things like `http.method`, `http.url`, `http.host`, etc. Should we keep these or should we make an effort to remove them?